### PR TITLE
:muscle: update dash-extensions

### PR DESCRIPTION
**Pull request by jonasvdd**
TODO:
- [ ] analyze why `dash-extensions` `1.0.1` appears to work and why `1.0.4` doesn't
- [ ] look into `use_reloader` argument of `Dash` `run_server`

### CODE REVIEW

File: examples/dash_apps/12_file_selector.py [hunk 1]
```diff
@@ -12,12 +12,11 @@
...
 import dash_bootstrap_components as dbc
 import plotly.graph_objects as go
-from dash import Input, Output, State, callback_context, dcc, html, no_update
-from dash_extensions.enrich import (
-    DashProxy,
-    ServersideOutput,
-    ServersideOutputTransform,
-)
+
+# from dash import Input, Output, State,
```
Comment (jvdd) [inline examples/dash_apps/12_file_selector.py:16]:
```suggestion
```

**Review by jvdd — APPROVED** [review]
LGTM!

**Comment by jonasvdd** [comment]
related #214

**Comment by jonasvdd** [comment]
related issue #262

**Pull request merged**